src/components/user/education/education.service.ts

```diff
@@ -43,7 +43,10 @@ export class EducationService {
     };
   }
 
-  async create(input: CreateEducation, session: ISession): Promise<Education> {
+  async create(
+    { userId, ...input }: CreateEducation,
+    session: ISession
+  ): Promise<Education> {
```

Comment:
How does the education node get connected to the user now?

Reply:
@CarsonF the education node gets connected to the user just a few lines deeper in this function:

```
    // connect the Education to the User.
    const query = `
      MATCH (user: User {id: $userId, active: true}),
        (education:Education {id: $id, active: true})
      CREATE (user)-[:education {active: true, createdAt: datetime()}]->(education)
      RETURN  education.id as id
      `;

    await this.db
      .query()
      .raw(query, {
        userId: session.userId,
        id,
      })
      .first();

    return await this.readOne(id, session);
```

Reply:
Doh! I see though that that is incorrectly using the session user Id. We want to attach to the user specified not the user making the call.

Reply:
Good catch! Fixed!